### PR TITLE
update(ssl): add troubleshooting section

### PR DESCRIPTION
**Pull request by marielle-spiteri**
* transfer old Support KB to official documentation as part of Learning project
* https://customer.bonitasoft.com/knowledgebase/display/4875
* https://bonitasoft.atlassian.net/browse/LT-190

### CODE REVIEW

File: modules/ROOT/pages/ssl.adoc [hunk 1]
```diff
@@ -168,3 +168,31 @@ Another feature of this valve is to replace the apparent scheme (http/https) and
...
+* You are using a self-signed certificate
+* You are using a certificate that has been issued by an internal certificate authority
+* Your certificate is outdated
+
+*Solutions:*  
+
+* Make sure you have imported the public certificate of the target instance into the truststore according to the Connecting to SSLServices instructions listed above  
+* Make sure any certificates have been imported into the correct truststore; you may have multiple JRE/JDKs  
+* Check to see that the correct truststore is in use. If `-Djavax.net.ssl.trustStore` has been configured, it will override the location of the default truststore, which will need to be checked
+* Check if your Antivirus tool has "SSL Scanning" blocking SSL/TLS. If it does, disable this feature or set exceptions for the target addresses
```
Comment (NathalieC) [inline modules/ROOT/pages/ssl.adoc]: Not sure we can recommend to disable the antivirus. Set exceptions seems more appropriate. I would only keep the second option.

**Review by NathalieC — APPROVED** [review]

File: modules/ROOT/pages/ssl.adoc (hunk 1 above)
Comment (abirembaut) [reply modules/ROOT/pages/ssl.adoc]: Maybe we can put it this way ?
```suggestion
* Check if your Antivirus tool has "SSL Scanning" blocking SSL/TLS. If it does, disable this feature temporarily or, preferably, set exceptions for the target addresses  
```

File: modules/ROOT/pages/ssl.adoc [hunk 2]
```diff
@@ -168,3 +168,31 @@ Another feature of this valve is to replace the apparent scheme (http/https) and
...
+* Your certificate is outdated
+
+*Solutions:*  
+
+* Make sure you have imported the public certificate of the target instance into the truststore according to the Connecting to SSLServices instructions listed above  
+* Make sure any certificates have been imported into the correct truststore; you may have multiple JRE/JDKs  
+* Check to see that the correct truststore is in use. If `-Djavax.net.ssl.trustStore` has been configured, it will override the location of the default truststore, which will need to be checked
+* Check if your Antivirus tool has "SSL Scanning" blocking SSL/TLS. If it does, disable this feature or set exceptions for the target addresses  
+* Verify that the target server is configured to serve SSL correctly. This can be done with an SSL Server Test tool
+* Make sure you used keytool to install the new certificate. More info online: http://docs.oracle.com/javase/8/docs/technotes/tools/windows/keytool.html
```
Comment (abirembaut) [inline modules/ROOT/pages/ssl.adoc]:
```suggestion
* Make sure you used keytool to install the new certificate. More info http://docs.oracle.com/javase/8/docs/technotes/tools/windows/keytool.html[online].
```

**Review by abirembaut — COMMENTED** [review]
LGTM
Just a few suggestions

File: modules/ROOT/pages/ssl.adoc (hunk 2 above)
Comment (marielle-spiteri) [reply modules/ROOT/pages/ssl.adoc]: I approve both suggestions, do I need to click on commit sugggestion then commit changes?

**Pull request merged**